codalab/model/bundle_model.py

```diff
@@ -1106,7 +1106,8 @@ def update_bundle(self, bundle, update, connection=None):
         # Generate a list of metadata keys that will be deleted and udpate metadata key-value pair
         metadata_delete_keys = []
         for key, value in metadata_update.items():
-            if value is None:
+            # Delete the key,value pair when the delete flag is set to True and value is None
+            if delete and value is None:
```

Comment:
We are changing the default behavior from `delete=True` to `delete=False`...are there any other calls to `update_bundle` that we need to update?

Reply:
No... `delete` operation is only introduced when we were trying to add the metadata field `staged_status`, as this field will disappear once the bundle successfully gets assigned to a worker. We didn't have this before, so no other functions or features are affected.

Reply:
The previous code is equivalent to `delete=True`, but now the default is `delete=False`.